perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -329,7 +330,7 @@ def RunForMinutes(proc, mins_to_run, mins_per_call):
 
 def GetConfig(user_config):
   config = configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
-  if FLAGS.against_device or FLAGS.prefill_device:
+  if FLAGS.fio_target_mode != 'against_file_without_fill':
```

Comment:
Not done? Need to define the string 'against_file_without_fill' as a constant at the top of this file and only refer to the constant in subsequent code.